src/util.cpp

```diff
@@ -33,6 +33,32 @@ int greater(double x, double y) {
     return x > y;
 }
 
+double sign(double x) {
```

Comment:
You can make use of the greater or less function here.

Reply:
Yes, it can be used. I just wanted it to be straightforward for now. Will change it...

Reply:
I was thinking wrong about this.. I think it makes sense to have it as `return (x >= 0.0) ? 1.0 : -1.0`. In case `x == 0.0` the sign should be positive I believe..